lib/foo.js

```diff
@@ -2,7 +2,9 @@ var service = require('./service');
 
 exports.bar = function (context, event) {
   if (!event.size) {
+    console.log('Missing parameter: size');
     context.fail('Missing parameter: size');
+    return;
```

Comment:
Great, but if context.fail raises an exception, should I wrap the app.handler call on a try/catch block?